src/braket/aws/aws_quantum_job.py

```diff
@@ -64,22 +64,22 @@ def create(
         cls,
         device: str,
```

Comment:
This needs to be `str | None` as this parameter is explicitly allowed to be None, as mentioned in the doc string